wasm/wat/lexer_test.go

```diff
@@ -571,9 +647,9 @@ func BenchmarkLex(b *testing.B) {
 		b.Run(bm.name, func(b *testing.B) {
 			b.ReportAllocs()
 			for i := 0; i < b.N; i++ {
-				err := lex(noopParseToken, "test.wasm", bm.data)
+				line, col, err := lex(noopParseToken, bm.data)
 				if err != nil {
-					panic(err)
+					panic(fmt.Errorf("%d:%d: %w", line, col, err))
```

Comment:
a real caller who knows the filename could put that in the format, also!

Reply:
💯